stdlib/printexc.mli

```diff
@@ -15,6 +15,9 @@
 
 (** Facilities for printing exceptions and inspecting current call stack. *)
 
+type nonrec exn = exn = ..
```

Comment:
Is the idea at some point to rename `Printexc` to `Exn` (and keep `Printexc` as a deprecated alias)?  If so, it would make sense to call the type `t`.

Reply:
That seems like a good long-term plan.  I've renamed the type `t`.